web/conf/context.jsonld

```diff
@@ -363,8 +361,7 @@
       "@id" : "https://d-nb.info/standards/elementset/gnd#SubjectHeading"
     },
     "note" : {
-      "@id" : "http://id.loc.gov/ontologies/bibframe/note",
-      "@container": "@set"
```

Comment:
Declaring @conatiner as @set was correct, it shouldn't be removed!
I am admittedly confused to see the tests build succesfully nonetheless :(
Shoudl be further investigated in a new issue.

Reply:
Ah, I see: web/conf/context.jsonld is build automatically using the context-labels.json. And the latter wasn't updated since end of 2019.  But web/conf/context.jsonld was updated multiple times then. This is not good because changing web/conf/context.jsonld will be overriden by context-labels.json. Phew. Will make an issue.

Reply:
Thanks for the reminder. I and obviously you too, totally forgot how it works.